### PR TITLE
JAX: Fix burning in the same dropout rng

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*
@astonzhang We should run a clean build for this change since it affects all the notebooks with dropout.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_linear-regression/oo-design.md [hunk 1]
```diff
@@ -292,6 +292,12 @@ You may notice that `Module` is a subclass of `tf.keras.Model`, the base class o
 It provides convenient features to handle neural networks. For example, it invokes the `call` method in the built-in `__call__` method. Here we redirect `call` to the `forward` function, saving its arguments as a class attribute. We do this to make our code more similar to other framework implementations.
 :end_tab:
 
+:begin_tab:`jax`
+You may notice that `Module` is a subclass of `linen.Module`, the base class of neural networks in Flax.
+It provides convenient features to handle neural networks. For example, it handles the model parameters, provides `nn.compact` decorator to simplify code, invokes the `__call__` method among other things.
```
Comment (astonzhang) [inline chapter_linear-regression/oo-design.md:297]:
```suggestion
It provides convenient features to handle neural networks. For example, it handles the model parameters, provides the `nn.compact` decorator to simplify code, invokes the `__call__` method among other things.
```

File: chapter_linear-regression/oo-design.md [hunk 2]
```diff
@@ -292,6 +292,12 @@ You may notice that `Module` is a subclass of `tf.keras.Model`, the base class o
...
+Here we also redirect `__call__` to the `forward` function. We do this to make our code more similar to other framework implementations.
```
Comment (astonzhang) [inline chapter_linear-regression/oo-design.md:298]:
```suggestion
Here we also redirect `__call__` to the `forward` method. We do this to make our code more similar to other framework implementations.
```

**Review by astonzhang — APPROVED** [review]

**Comment by astonzhang** [comment]
clean build: http://ci.d2l.ai/blue/organizations/jenkins/d2l-en/detail/master/553/pipeline/
